epinar/assignment5/unit_tests

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import unittest
+from unknown_alphabet import find_alphabets, find_alphabet
+
+class TestDictionary(unittest.TestCase):
+
+	def check_equal(self, L1, L2):
+		return sorted(L1) == sorted(L2)
```

Comment:
I love this function. It is very convenient and I think we should use it in the tests for the final solution.

Reply:
Definitely helpful!